package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "start": "node src/server.js",
     "dev": "npm run swagger-autogen && cross-env NO_CLI=true nodemon src/server.js",
-    "cli": "nodemon src/server.js --database foobar.db",
+    "cli": "nodemon src/server.js --database chinook.db --port 8000 createsuperuser  --username john --password 123456789",
```

Comment:
Hello @AbegaM - is it right for us to amend the main startup command for Soul?  I'm expecting our capability to be optional. Would it be better to create a new command?  Maybe `cli-auth`?